.github/workflows/quickstart_helloworld_mutual_ssl.yml

```diff
@@ -0,0 +1,15 @@
+name: WildFly helloworld-mutual-ssl Quickstart CI
```

Comment:
The filename should be "[quickstart_helloworld-mutual-ssl_ci.yml" and this Quickstart actually requires additional functionality in the abstract reused quickstart_ci.yml, which I will add later,for now please use the following code:

```
name: WildFly helloworld-mutual-ssl Quickstart CI

on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
    paths:
      - helloworld-mutual-ssl/**'
      
# Only run the latest job
concurrency:
  group: '${{ github.workflow }} @ ${{ github.ref || github.run_id }}'
  cancel-in-progress: true

jobs:
  Test-build-default-matrix:
    name: BUILD DEFAULT - JDK${{ matrix.jdk }} - ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        jdk: [11, 17]
        os: [ubuntu-20.04, windows-latest]
    steps:
      - uses: actions/checkout@v4
        with:
          path: quickstarts
      - name: Set up JDK ${{ matrix.jdk }}
        uses: actions/setup-java@v3
        with:
          java-version: ${{ matrix.jdk }}
          distribution: 'temurin'
          cache: 'maven'
      - name: Build ${{ inputs.QUICKSTART_PATH }} Quickstart for Release
        run: |
          cd quickstarts
          cd helloworld-mutual-ssl
          mvn -U -B -fae clean install -Drelease
        shell: bash
      - name: Build, run & test helloworld-mutual-ssl Quickstart with provisioned-server profile
        run: |
          cd quickstarts
          cd helloworld-mutual-ssl
          echo "Building provisioned server..."
          mvn -U -B -fae clean package -Pprovisioned-server
          echo "Starting provisioned server..."
          mvn -U -B -fae wildfly:start -DjbossHome=target/server -Dstartup-timeout=120
#          echo "Testing provisioned server..."
#          mvn -U -B -fae verify -Dserver.host=https://localhost:8080 -Pintegration-testing 
          echo "Shutting down provisioned server..."
          mvn -U -B -fae wildfly:shutdown
        shell: bash
      - name: Build helloworld-mutual-ssl Quickstart with openshift profile
        run: |
          cd quickstarts
          cd helloworld-mutual-ssl
          mvn -U -B -fae clean package -Popenshift
        shell: bash
      - uses: actions/upload-artifact@v3
        if: failure()
        with:
          name: surefire-reports-JDK${{ matrix.jdk }}-${{ matrix.os }}
          path: 'quickstarts/helloworld-mutual-ssl/**/surefire-reports/*.txt'

  Test-build-with-deps-matrix:
    name: BUILD WITH DEPS - JDK${{ matrix.jdk }} - ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        jdk: [11, 17]
        os: [ubuntu-20.04, windows-latest]
    steps:
      - uses: actions/checkout@v4
        with:
          repository: wildfly/wildfly
          ref: ${{ github.base_ref }}
          path: wildfly
      - uses: actions/checkout@v4
        with:
          path: quickstarts
      - name: Set up JDK ${{ matrix.jdk }}
        uses: actions/setup-java@v3
        with:
          java-version: ${{ matrix.jdk }}
          distribution: 'temurin'
          cache: 'maven'
      - name: Build Server
        run: |
          cd wildfly
          mvn -U -B -fae -DskipTests clean install
        shell: bash
      - name: Get Server Version
        run: |
          cd wildfly
          echo "VERSION_SERVER=$(mvn -N org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_ENV
        shell: bash
      - name: Build Quickstart for Release with built Server version
        run: |
          cd quickstarts
          cd helloworld-mutual-ssl
          mvn -U -B -fae clean package -Drelease -Dversion.server=${{ env.VERSION_SERVER }}
        shell: bash
      - name: Build, run & test helloworld-mutual-ssl Quickstart with provisioned-server profile, and built Server version
        run: |
          cd quickstarts
          cd helloworld-mutual-ssl
          echo "Building provisioned server..."
          mvn -U -B -fae clean package -Pprovisioned-server -Dversion.server=${{ env.VERSION_SERVER }}
          echo "Starting provisioned server..."
          mvn -U -B -fae wildfly:start -DjbossHome=target/server -Dstartup-timeout=120
#          echo "Testing provisioned server..."
#          mvn -U -B -fae verify -Dserver.host=http://localhost:8080 -Pintegration-testing
          echo "Shutting down provisioned server..."
          mvn -U -B -fae wildfly:shutdown
        shell: bash
      - name: Build helloworld-mutual-ssl Quickstart with openshift profile, and built Server version
        run: |
          cd quickstarts
          cd helloworld-mutual-ssl
          mvn -U -B -fae clean package -Popenshift -Dversion.server=${{ env.VERSION_SERVER }}
        shell: bash
      - uses: actions/upload-artifact@v3
        if: failure()
        with:
          name: surefire-reports-JDK${{ matrix.jdk }}-${{ matrix.os }}
          path: 'quickstarts/**/surefire-reports/*.txt'
```

Reply:
fixed!

Reply:
all features this QS needs are now available, so please change this file to:

```
name: WildFly helloworld-mutual-sslQuickstart CI

on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
    paths:
      - 'helloworld-mutual-ssl/**'
      - '.github/workflows/quickstart_ci.yml'

jobs:
  call-quickstart_ci:
    uses: ./.github/workflows/quickstart_ci.yml
    with:
      QUICKSTART_PATH: helloworld-mutual-ssl
      SERVER_PROVISIONING_SERVER_HOST: https://localhost:8443/
      TEST_PROVISIONED_SERVER: true
      TEST_OPENSHIFT: false
```

Reply:
updated!

Reply:
for the ci, we may need to add the path to server config directory using the -Djboss.server.config.dir=path/to/standalone/configuration param. Would this be part of this script?